core/dbt/tests/tables.py

```diff
@@ -0,0 +1,339 @@
+from dbt.tests.util import run_sql
+from dbt.context import providers
+from unittest.mock import patch
+from contextlib import contextmanager
+
+
+class TableComparison:
```

Comment:
This is not "new way of testing". This is "old way of testing" copy-pasted to the new way so that we can convert the tests without so much pain. Ideally it would be converted to use macros and simplified a great deal. Eventually.

Reply:
Alright, let's forget about this class for now